wiki/Play_style/Double_tapping/en.md

```diff
@@ -11,7 +11,7 @@ tags:
 
 *Not to be confused with [Alternating](/wiki/Play_style/Alternating).*
 
-**Double tapping** is a tapping style for [osu!standard](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by pressing down on both of their set keys on their keyboard (`Z` and `X` by default). This style is most often used as a temporary substitute tapping style for extremely fast [stacks](/wiki/Mapping_Techniques/Stack) and [streams](/wiki/Beatmap/Pattern/Stream) that are otherwise too difficult for some players. This style is known to be the least accurate.
+**Double tapping** is a tapping style for [osu!](/wiki/Game_mode/osu!) game mode in which players click circles, slide sliders, and spin spinners by pressing down on both of their set keys on their keyboard (`Z` and `X` by default). This style is most often used as a temporary substitute tapping style for extremely fast [stacks](/wiki/Mapping_Techniques/Stack) and [streams](/wiki/Beatmap/Pattern/Stream) that are otherwise too difficult for some players. This style is known to be the least accurate.
```

Comment:
```suggestion
**Double tapping** is a tapping style for [osu!](/wiki/Game_mode/osu!) in which players click circles, slide sliders, and spin spinners by pressing down on both of their set keys on their keyboard (`Z` and `X` by default). This style is most often used as a temporary substitute tapping style for extremely fast [stacks](/wiki/Mapping_Techniques/Stack) and [streams](/wiki/Beatmap/Pattern/Stream) that are otherwise too difficult for some players. This style is known to be the least accurate.
```